cosmotech/coal/cli/commands/s3_bucket_loader.py

```diff
@@ -34,28 +30,96 @@ def get_connection(is_client=True):
               type=str,
               show_envvar=True,
               required=True)
+@click.option("--prefix",
+              "file_prefix",
```

Comment:
Maybe change the name to "prefix_filter" to not confuse with a added prefix to file at upload.